Burn Bryte by Roll20/BurnBryte.html

```diff
@@ -1149,295 +1129,231 @@ <h3 data-i18n="special-a">Special Abilities</h3>
-            update["movement_speed"] = page.data["Speed"];
-            update["size"] = page.data["Size"];
-            update["shield_health_level"] = page.data["Shield Health Levels"];
-            update["shield_health_level_max"] = page.data["Shield Health Levels"];
-
-            var abilityArray = JSON.parse(page.data["data-Ability"]);
-            abilityArray.forEach(function(ability) {
-                var id = generateRowID();
-                update["repeating_abilities_" + id + "_ability"] = ability.Ability;
-                update["repeating_abilities_" + id + "_description"] = ability.Description;
-                update["repeating_abilities_" + id + "_special_ability_flag"] = "false";
-            });
-
-            var attackArray = JSON.parse(page.data["data-Attacks"]);
-            attackArray.forEach(function(attack) {
-                var id = generateRowID();
-                update["repeating_attacks_" + id + "_attack"] = attack.Attack;
-                update["repeating_attacks_" + id + "_complexity"] = attack.Complexity;
-                update["repeating_attacks_" + id + "_details"] = attack.Hit;
-                update["repeating_attacks_" + id + "_range"] = attack.Range;
-                update["repeating_attacks_" + id + "_type"] = attack.Type;
-                update["repeating_attacks_" + id + "_attacks_flag"] = "false";
-            });
-
-            if (page.data["data-Equipment"]) {
-                var equipmentArray = JSON.parse(page.data["data-Equipment"]);
-
-                getCompendiumPage(equipmentArray, function(items) {
-                    makeItem(items);
+   var update = {};
+   var id = generateRowID();
+   getAttrs(["drop_name", "drop_data", "drop_content"], function(v) {
+       try {
+           pagedata = JSON.parse(v.drop_data);
+       } catch(e) {
+           pagedata = v.drop_data;
+       };
+       var page = {
+           name: v.drop_name,
+           data: pagedata,
+           content: v.drop_content
+       };
+       //Handle updated page data and included events, abilities, scores, etc, based on the type of page dropped in
+       var category = page.data["Category"];
+       switch(category) {
+           case "Culture":
+               (page.data["Type"] === "Borders") ? update["culture_border"] = page.name :
+               (page.data["Type"] === "Diversity") ? update["culture_diversity"] = page.name :
+               (page.data["Type"] === "Economic Status") ? update["culture_economic"] = page.name :
+               update["culture_population"] = page.name;
+               break;
+           case "Lore":
+               if (page.data["data-Type"] === "Culture") {
+                   update["culture"] = page.name;
+               };
+               break;
+           case "Species":
+               update["species"] = page.name;
+               break;
+           case "Story Paths":
+               update["repeating_story_" + id + "_story"] = page.name;
```

Comment:
Update all of these sort of strings to use [interpolation](https://en.wikipedia.org/wiki/String_interpolation#JavaScript) so they will be easier to read.